src/opnsense/scripts/OPNsense/CaptivePortal/process_accounting_messages.php

```diff
@@ -82,14 +83,14 @@
                 $stmt->bindParam(':sessionid', $row['sessionid']);
                 $stmt->execute();
                 if (method_exists($authenticator, 'stopAccounting')) {
-                    $time_spend = time() - $row['created'];
-                    $authenticator->stopAccounting($row['username'], $row['sessionid'], $time_spend, $row['bytes_in'], $row['bytes_out'], $row['ip_address']);
+                    $time_spend = time() - intval($row['created']);
```

Comment:
FWIW, this was done to fix `PHP Deprecated:  Implicit conversion from float 459.6189193725586 to int loses precision in /usr/local/opnsense/mvc/app/library/OPNsense/Auth/Radius.php on line 383`